v2/go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/spiffe/go-spiffe/v2
 
-go 1.19
+go 1.21
+
+toolchain go1.22.1
```

Comment:
This was auto-added by `go mod tidy` when run with Go v1.22.1. I can remove it if it is going to cause any backward compatibility issues and still compatible with latest tooling.